db/models.py

```diff
@@ -50,3 +54,51 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
```

Comment:
Ensure that the `show_time` is formatted correctly in the `__str__` method of the `MovieSession` model. Consider using Django's date formatting utilities for consistent output.